lang/en.json

```diff
@@ -1,22 +1,28 @@
 {
-  "ACTOR.TypeCustos": "Custos",
-  "ATTRIBUTE.HitPoints": "Hit Points",
-  "ITEM.TypeMeleeWeapon": "Melee Weapon",
-  "LexArcana.Name": "Name",
-  "LexArcana.Attributes": "Attributes",
-  "LexArcana.Biography": "Biography",
-  "LexArcana.HitPoints": "Biography",
-  "LexArcana.Pietas": "Pietas",
-  "LexArcana.Inventory": "Inventory",
-  "LexArcana.Rituals": "Rituals",
-  "LexArcana.Features": "Features",
-  "LexArcana.ItemName": "ItemName",
-  "LexArcana.Encumbrance": "Encumbrance",
-  "LexArcana.Protection": "Protection",
-  "LexArcana.Damage": "Damage",
-  "LexArcana.Range": "Range",
-  "LexArcana.Difficulty": "Difficulty",
-  "LexArcana.Feat": "Feat",
-  "LexArcana.Parry": "Parry"
+  "ACTOR.TypeCustos":             "Custos",
+  "ATTRIBUTE.HitPoints":          "Hit Points",
+  "ITEM.TypeMeleeWeapon":         "Melee Weapon",
+  "LexArcana.PeritiaDeBello":     "De Bello",
+  "LexArcana.PeritiaDeNatura":    "De Natura",
+  "LexArcana.PeritiaDeSocietate": "De Societate",
+  "LexArcana.PeritiaDeMagia":     "De Magia",
+  "LexArcana.PeritiaDeScientia":  "De Scientia",
+  "LexArcana.PeritiaDeCorpore":   "De Corpore",
```

Comment:
De Corpore is a value for a peritia, thus it should be named like that: ```LexArcana.Peritia.DeCorpore```. The same goes for all peritiae

Reply:
Why not, I was following examples around, I'll take a look :)